src/khoj/utils/rawconfig.py

```diff
@@ -41,6 +41,10 @@ def input_filter_or_files_required(cls, input_filter, values, **kwargs):
         return input_filter
 
 
+class PageContentConfig(TextConfigBase):
+    input_files: Optional[List[str]]
```

Comment:
For this data type, the input is not optional.
```suggestion
    input_pages: List[str]
```